src/Corax/Queries/TermProviders/TermProvider.Range.cs

```diff
@@ -184,13 +184,14 @@ public bool Next(out TermMatch term)
             {
                 goto Empty;
             }
-
+            
             var termId = _iterator.CreateReaderForCurrent().ReadLittleEndianInt64();
-            term = _searcher.TermQuery(termId);
+            // Ratio will be always 1 (sizeof(T)/sizeof(T))
```

Comment:
What is the ratio _for_?

Reply:
Term length in comparison to the average length of the term under the specific field.
In the case of numbers (when we use Longs/Doubles trees) our terms have the same length (sizeof(T))

Reply:
Okay, let's add that as a comment, so it will be clear in a year or two

Reply:
Include link to the document (probably we should add it somewhere in our own public knowledge base) and the citation to the location in the document where this is explained.